scripts/hid_configurator/configurator.py

```diff
@@ -263,38 +319,80 @@ def perform_dfu(dev, args):
     img_length = os.stat(dfu_image).st_size
     print('DFU image size: {} bytes'.format(img_length))
 
+    for i in range(3):
+        dfu_info = dfu_sync(dev, args)
+        if dfu_info is not None:
+            if dfu_info[0] != 0:
+                time.sleep(1)
+            else:
+                break
+
+    if dfu_info is None:
+        print('Cannot start DFU, device not responding')
+        return
+
+    if dfu_info[0] != 0:
+        print('Cannot start DFU, already in progress')
+        return
+
+    img_csum = file_crc(dfu_image)
+    if (dfu_info[1] == img_length) and (dfu_info[2] == img_csum) and (dfu_info[3] <= img_length):
+        print('Resume DFU at {}'.format(dfu_info[3]))
+        offset = dfu_info[3]
+    else:
```

Comment:
Add print to indicate that we start DFU ?

Reply:
I don't think it is needed as progress bar indicates that